.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2
+
+aliases:
+  - &defaults
+    docker:
+      - image: circleci/node:6
```

Comment:
Would it be worth testing on multiple Node versions? Since this is a library that can be used outside of of Apiary.

Reply:
I'd leave that on an organic path. If there are users then it is easy to add.

Rather we might want to move away from v6 to v8 as that is the current LTS.